core/src/main/scala/org/broadinstitute/dsp/workbench/hamm/service/CostService.scala

```diff
@@ -41,5 +41,6 @@ class CostService(samAuthProvider: SamAuthProvider, dbRef: DbReference, jobTable
   }
 }
 
-final case class WorkflowCostResponse(workflowId: WorkflowId, cost: Double)
-final case class JobCostResponse(jobId: JobId, cost: Double)
+trait ServiceResponse
```

Comment:
What's the point of having this `trait`?

Reply:
Oops - left that in after deleting a thing I was trying